.github/benchmark-files/benchmark.ini

```diff
@@ -0,0 +1,11 @@
+max_execution_time=0
+error_reporting=0
```

Comment:
We remove all error handling, printing and displaying so that it does not interfere with the tests